Classes/Domain/Repository/JobRepository.php

```diff
@@ -32,6 +33,32 @@ public function findAll()
         )->execute();
     }
 
+    /**
+     * Returns a job list matching the current user.
```

Comment:
Maybe _Returns a list of jobs where a given user has contributed._ Currently this only becomes clear when reading the code.